.travis.yml

```diff
@@ -1,14 +1,14 @@
 language: scala
 scala:
-  - 2.10.3
+  - 2.12.1
 
 script: sbt ++$TRAVIS_SCALA_VERSION test -Dloglevel=debug
 
 before_install: gem install fluentd
 
 jdk:
- - openjdk7
- - oraclejdk7
+ - openjdk8
```

Comment:
Yes! Found this out working with another PR. Will update this one with the correct build configuration.